src/mono/sample/iOS/Makefile

```diff
@@ -6,6 +6,7 @@ AOT?=false
 TARGET?=iossimulator
 DEPLOY_AND_RUN?=true
 APP_SANDBOX?=false
+STRIP_DEBUG_SYMBOLS?=true // only used when measuring SOD via build-appbundle make target
```

Comment:
Since this is a new feature, dotnet/perf does not know about, it should not be turned on by default.

Reply:
Good point. Is it considered as a default behavior or a separate measurement?

Reply:
I think for start it will be considered as a separate measurement.